common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -1958,16 +1958,24 @@ def ensure_indexes(self):
                 ('_id.name', pymongo.ASCENDING),
                 ('_id.revision', pymongo.ASCENDING),
             ],
-            background=True)
+            # While this index would more safely be "background=True", it's not currently created that way.
```

Comment:
So if we, or someone, had a production environment without any of these indices, they would be created in the foreground, correct?